src/windows/functions.cpp

```diff
@@ -393,6 +393,37 @@ MOUNTUTILS_RESULT EjectDriveLetter(TCHAR driveLetter) {
   return MOUNTUTILS_SUCCESS;
 }
 
+BOOL IsDriveLetterEjectable(TCHAR driveLetter) {
```

Comment:
Sorry, didn't mean to open up a can of worms - when I left my comment I absentmindedly forgot that we're mixing languages here ;-)